mysql-compatibility.md

```diff
@@ -93,171 +76,118 @@ mysql> select _tidb_rowid, id from t;
 
 ### Performance schema
 
-Performance schema tables return empty results in TiDB. TiDB uses a combination of [Prometheus and Grafana](/monitor-a-tidb-cluster.md) for performance metrics instead.
-
-TiDB supports the `events_statements_summary_by_digest` table from TiDB 3.0.4. For more information, see [Statement Summary Table](/statement-summary-tables.md).
+TiDB uses a combination of Prometheus and Grafana to store and query the performance monitoring metrics. The Performance schema is an empty schema.
 
 ### Query Execution Plan
 
-The output format of Query Execution Plan (`EXPLAIN`/`EXPLAIN FOR`) in TiDB is greatly different from that in MySQL. Besides, the output content and the privileges setting of `EXPLAIN FOR` are not the same as those of MySQL. See [Understand the Query Execution Plan](/query-execution-plan.md) for more details.
+The output format of Query Execution Plan (`EXPLAIN`/`EXPLAIN FOR`) in TiDB is greatly different from that in MySQL. In addition, the output content and the privileges setting of `EXPLAIN FOR` are also different from those of MySQL. See [Understand the Query Execution Plan](/query-execution-plan.md) for more details.
 
 ### Built-in functions
 
 TiDB supports most of the MySQL built-in functions, but not all. See [TiDB SQL Grammar](https://pingcap.github.io/sqlgram/#functioncallkeyword) for the supported functions.
 
 ### DDL
 
-In TiDB DDL does not block reads or writes to tables while in operation. However, some restrictions currently apply to DDL changes:
-
 + Add Index:
-    - Does not support creating multiple indexes at the same time.
-    - Does not support the `VISIBLE/INVISIBLE` index.
-    - Other Index Type (HASH/BTREE/RTREE) is supported in syntax, but not applicable.
+    - Does not support creating multiple indexes using a single SQL statement.
+    - Supports creating index of different types (HASH/BTREE/RTREE) only in syntax, but not applicable.
+    - Supports the `VISIBLE`/`INVISIBLE` index and ignores other options.
```

Comment:
I didn't see this line in [the latest Chinese version](https://pingcap.com/docs-cn/stable/mysql-compatibility/#ddl-%E7%9A%84%E9%99%90%E5%88%B6). @tangenta Please also take a look.
```suggestion
```

Reply:
dev and 4.0 versions are different. See https://github.com/pingcap/docs/pull/2790#issuecomment-643077418. @lilin90

Reply:
@yikeke Got it. Thanks!